src/View/Score.elm

```diff
@@ -0,0 +1,53 @@
+module View.Score exposing (view)
+
+import Svg exposing (Svg, circle, g, svg)
+import Svg.Attributes exposing (cx, cy, fill, height, r, stroke, strokeWidth, style, viewBox, width)
+
+
+scoreMarker : String -> Int -> Int -> Svg msg
+scoreMarker color y x =
+    let
+        baseAttrs : List (Svg.Attribute msg)
+        baseAttrs =
+            [ cx (toString x)
+            , cy (toString y)
+            , r "5.5%"
+            , stroke color
+            , fill "none"
+            , strokeWidth "1.5%"
+            , width "100"
+            , height "100"
+            ]
+
+        makeCircle : List (Svg.Attribute msg) -> Svg msg
+        makeCircle attrs =
+            Svg.circle baseAttrs []
+    in
+    circle baseAttrs []
```

Comment:
The duplication from `View.Occupant` is fine, but I think we could avoid it. We could, say, add a `View.Marker` module or add a view function to `Marker` itself. let's say we extend the `Marker` module:

```elm
module Marker exposing (Marker(..))

import Svg exposing (Svg)

type Marker = Ring | Disc

type alias Scale = Float
type alias Coordinate = (Float, Float)
type alias Color = String

view : Radius -> Coordinate -> Color -> Marker -> Svg msg
```